qutebrowser/browser/qutescheme.py

```diff
@@ -425,6 +425,17 @@ def qute_settings(url):
     return 'text/html', html
 
 
+@add_handler('back')
+def qute_back(url):
+    """Handler for qute://back.
+
+    Simple page to free ram / lazy load a site, goes back on focusing the tab.
+    """
+    html = jinja.render('back.html',
+                        title='Suspended: ' + url.url().split('#')[-1])
```

Comment:
You should be able to do [`url.fragment()`](https://doc.qt.io/qt-5/qurl.html#fragment) here.